lib/control-connection.js

```diff
@@ -12,6 +12,7 @@ const requests = require('./requests');
 const utils = require('./utils');
 const types = require('./types');
 const f = util.format;
+const VersionNumber = types.VersionNumber;
```

Comment:
+1 for VersionNumber. It might be "too much" for now, but on the long run it will payoff.